omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -691,7 +690,10 @@
 # allowable choices come from the PGSSLMODE environment var for libpq.
 # supported sslmode values: disable | require
 default['private_chef']['postgresql']['sslmode'] = 'disable'
-
+# for now, copy the nginx certs by hand into the postgres data directory, chmod and chown
+# later, this will be automated and these comments will be removed
+default['private_chef']['postgresql']['ssl_cert_file'] = "#{default['private_chef']['postgresql']['dir']}/data/#{node['fqdn']}.crt"
```

Comment:
you might have to use the node attributes instead.
`default['private_chef']['postgresql']['ssl_cert_file'] = node['private_chef']['postgresql']['data_dir']/#{node['fqdn']}.crt` (assuming that is the name of the cert)

Reply:
Done.